### PR TITLE
auth controller created

### PR DESCRIPTION
**Pull request by emmebravo**
- we imported Router bc line 60 was throwing an error. we aren't sure if that's the best way to handle that if-statement. let us know how to handle it--whether it's a middleware or its own method.

### CODE REVIEW

File: packages/server/controllers/auth.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    );
+    user.save();
+    console.log(user);
+    res.status(200).json({ message: "Password updated successfully" });
+
+    if (process.env.NODE_ENV !== "production") {
+      router.get("/get-user/:id", async (req, res) => {
+        res.json(await User.findOne({ _id: req.params.id }));
+      });
+    }
```
Comment (indifferentghost) [inline packages/server/controllers/auth.js:64]: This is a dev route - in `authRoutes.js` have:
```js
if (process.env.NODE_ENV !== "production") {
  router.get("/get-user/:id", authController.getUser);
}
```

Comment (indifferentghost) [reply packages/server/controllers/auth.js:64]: This would be `getUser: async (req, res) => { /* code */ }`

**Review by indifferentghost — CHANGES_REQUESTED** [review]
Looks good except for the dev route.

File: packages/server/controllers/auth.js (hunk 1 above)
Comment (emmebravo) [reply packages/server/controllers/auth.js:64]: ah, ok, got it...

**Comment by emmebravo** [comment]
> Looks good except for the dev route.

fixed it :)

**Comment by emmebravo** [comment]
fixed dev route

**Review by indifferentghost — APPROVED** [review]

**Pull request merged**